src/api/app/models/workflow_run_filter.rb

```diff
@@ -0,0 +1,28 @@
+class WorkflowRunFilter
```

Comment:
Phew.... A PORO to rename params instead of renaming the params?

Reply:
https://github.com/openSUSE/open-build-service/pull/15979/commits/e21d15c73456017604f3c759f7ba7615e4ebacf4 one step toward dismissing it

Reply:
Renaming the params won't change anything... This object is adapting the way we use the params: our filter form sets parameters like `succeeded=1&running=1` and the way we consume the filters expects something like `status=['success','fail']`. We need some code to translate it.

Reply:
> We need some code to translate it.

Okay but for sure not a PORO you instantiate...

Reply:
I got rid of this class